src/test/Components/CreateImageWizard/steps/Repositories/Repositories.test.tsx

```diff
@@ -225,45 +360,126 @@ describe('Repositories edit mode', () => {
     const expectedRequest = repositoriesCreateBlueprintRequest;
     await waitFor(() => expect(receivedRequest).toEqual(expectedRequest));
   });
-  //  test(
-  //    'modal pops up when deselecting previously used repository',
-  //    { retry: 3 },
-  //    async () => {
-  //      const id = mockBlueprintIds['repositories'];
-  //      await renderEditMode(id);
-  //
-  //      const customRepositories = await screen.findByRole('button', {
-  //        name: /Custom repositories/,
-  //      });
-  //
-  //      user.click(customRepositories);
-  //
-  //      await screen.findByText(
-  //        /Removing previously added repositories may lead to issues with selected packages/i
-  //      );
-  //
-  //      const selectedRepositories = await screen.findByRole('button', {
-  //        name: /Selected repositories/,
-  //      });
-  //      user.click(selectedRepositories);
-  //
-  //      const repoCheckbox = await screen.findByRole('checkbox', {
-  //        name: /select row 0/i,
-  //      });
-  //      await waitFor(() => expect(repoCheckbox).toBeChecked());
-  //
-  //      user.click(repoCheckbox);
-  //      await screen.findByText(/Are you sure?/);
-  //      const removeAnywayBtn = await screen.findByRole('button', {
-  //        name: /Remove anyway/,
-  //      });
-  //      user.click(removeAnywayBtn);
-  //
-  //      await waitFor(() =>
-  //        expect(screen.queryByText(/Are you sure?/)).not.toBeInTheDocument()
-  //      );
-  //
-  //      await waitFor(() => expect(repoCheckbox).not.toBeChecked());
-  //    }
-  //  );
+
+  test('modal pops up when deselecting previously used repository', async () => {
+    const id = mockBlueprintIds['repositories'];
+    await renderEditMode(id);
+
+    const customRepositories = await screen.findByRole('button', {
+      name: /Custom repositories/,
+    });
+
+    user.click(customRepositories);
+
+    await screen.findByText(
+      /Removing previously added repositories may lead to issues with selected packages/i
+    );
+
+    await toggleSelected();
+
+    const repoCheckbox = await getFirstRepoCheckbox();
+    await waitFor(() => expect(repoCheckbox).toBeChecked());
+
+    user.click(repoCheckbox);
+    await screen.findByText(/Are you sure?/);
+    const removeAnywayBtn = await screen.findByRole('button', {
+      name: /Remove anyway/,
+    });
+    user.click(removeAnywayBtn);
+
+    await waitFor(() =>
+      expect(screen.queryByText(/Are you sure?/)).not.toBeInTheDocument()
+    );
+
+    await waitFor(() => expect(repoCheckbox).not.toBeChecked());
+  });
 });
+
```

Comment:
maybe for now we can move all the comment rows? if thats a cleanup pr :)

Reply:
I mean for now, if you move it to new file, IMOO it is better to move it without comments?

Reply:
I've removed the "On Recreate" tests.